network/network_integration_test.go

```diff
@@ -120,64 +113,12 @@ func TestNetworkIntegration_HappyFlow(t *testing.T) {
 	fmt.Printf("%v\n", node2.Diagnostics())
 }
 
-func TestNetworkIntegration_SignatureIncorrect(t *testing.T) {
```

Comment:
Why is this removed?

Reply:
because it didn't really test anything. For one node it manipulated the internals and for the other it didn't. Not really an integration test if internals are manipulated. Also with the new setup it's way harder to manipulate the used private key. You can now only do that from unit test context.